mscore/musescore.cpp

```diff
@@ -2315,11 +2315,12 @@ void MuseScore::reloadInstrumentTemplates()
 
 void MuseScore::askResetOldScorePositions(Score* score)
       {
-      if (score->mscVersion() < 300 && score->mscVersion() > 114) {
```

Comment:
It was an intention, because some scores crash on resetting positions which is not good, but MuseScore 3 doesn't support the scores created in MuseScore 1.X. The recommended way is to save the score in MuseScore 2.3.2 and open in MuseScore 3.

Reply:
I haven't seen it crashing on mine, but as we do have a 1.x importer we should rather fix that.

Reply:
I've just tried and opened lots of my 1.x scores, not just a handfull, but some 100: not a single crash on reseting positions